lib/gnat.ex

```diff
@@ -273,6 +353,15 @@ defmodule Gnat do
     %{state | receivers: receivers}
   end
 
+  defp process_message({:msg, topic, @request_sid, reply_to, body}, state) do
```

Comment:
How does `@request_sid` work as a method argument? Is it pattern matching on the variable in this module? Is that what we want?

Reply:
@film42 yes, this function clause will only match if you call it with a tuple that has `:msg` in the first position and `0` (`@request_sid`) in the third position. I think this makes sense because we want to segment any messages we have received for `sid=0` and handle them separately than other messages

Reply:
For some reason I was thinking `@request_sid` was a value that changed, but it's supposed to be a constant. That makes much more sense :+1:  Thanks for helping me learn :)